### PR TITLE
Add Goals 

### PR DESCRIPTION
**Pull request by ayuishii**
Add list of goals that have been discussed so far.

### CODE REVIEW

File: explainer.md [hunk 1]
```diff
@@ -52,17 +53,22 @@ goals of the project are, and a brief overview of how the solution works.
...
 
 
-## Goals [or Motivating use cases, or Scenarios]
-
-[What is the end-user need which this project aims to address?]
-
+## Goals
+- Allow web applications to evict partitions of data
+- Allow web developers to specify eviction ordering
+- Allow web developers to remove a service workers without clearing data for the entire origin
```
Comment (mkruisselbrink) [inline explainer.md:59]: I don't understand this one?

File: explainer.md [hunk 2]
```diff
@@ -52,17 +53,22 @@ goals of the project are, and a brief overview of how the solution works.
...
+- Allow web developers to reserve quota
+- Allow web developers to express performance, durability and other trade-off decisions on partitions of data
+- Ensure privacy across accounts on shared devices
```
Comment (mkruisselbrink) [inline explainer.md:62]: This goal seems to be the only one that is actually phrased as a "end-user need which this project aims to address". Not sure how to best phrase the end-user need for the rest, but probably something around making sure browsers don't delete data that users don't want to be deleted? (and I would definitely also keep what you have now; but also include what end-user needs we achieve by letting web developers/applications do these things).

Comment (ayuishii) [reply explainer.md:62]: Good point, I tried adding an additional point about preventing important data from being deleted. Let me know if it doesn't match what you had in mind. Thanks!

File: explainer.md (hunk 1 above)
Comment (ayuishii) [reply explainer.md]: I was trying to reword the use case for associating service worker with bucket to make entire service worker footprint eviction easy. I'll try re-phrasing a bit.

**Comment by pwnall** [comment]
Thank you!

This text is in fairly early stages, so we can fix it forward if we need to.

**Pull request merged**